lms/djangoapps/teams/serializers.py

```diff
@@ -112,7 +112,8 @@ class MembershipSerializer(serializers.ModelSerializer):
             view_name='teams_detail',
             read_only=True,
         ),
-        expanded_serializer=CourseTeamSerializer(read_only=True)
+        expanded_serializer=CourseTeamSerializer(read_only=True),
+        catch_expand_fields={'user'}
```

Comment:
I actually prefer `catch`, since it describes the fact that *this* field is to be expanded by *this* serializer, and not passed down. `exclude` makes me think that this field is to be ignored this level downwards.

It's a mouthful, but `expanded_fields_to_catch` would be less ambiguous.